engine/security/trust/content_trust.md

```diff
@@ -62,38 +62,19 @@ push replaces the last unsigned tag `latest` but does not affect the signed `lat
 The ability to choose which tags they can sign, allows publishers to iterate over
 the unsigned version of an image before officially signing it.
 
-Image consumers can enable DCT to ensure that images they use were
-signed. If a consumer enables DCT, they can only pull, run, or build
-with trusted images. Enabling DCT is like wearing a pair of
-rose-colored glasses. Consumers "see" only signed image tags and the less
-desirable, unsigned image tags are "invisible" to them.
+Image consumers can enable DCT to ensure that images they use were signed. If a 
+consumer enables DCT, they can only pull, run, or build with trusted images. 
+Enabling DCT is like wearing a pair of rose-colored glasses. Consumers "see" 
```

Comment:
I love the use of an analogy, but sometimes the rose-colored glasses can be seen as "bliss in ignorance" or ignoring red flags.
Perhaps something about a bouncer providing a first clearance for only trusted images getting into the node.  I'm not a fan of that, but maybe it helps get something to build from.